src/GlobalStyles.js

```diff
@@ -1,12 +1,23 @@
 import { createGlobalStyle } from 'styled-components/macro'
-
 import { getThemeColors } from './services/getThemeColors'
 
 export default createGlobalStyle`
   * {
     box-sizing: border-box;
   }
 
+  #root {
+    text-align: center;
+    display: grid;
+    grid-template-rows: ${({ userLoggedIn }) => userLoggedIn ? '110px' : '55px'} auto 55px;
```

Comment:
Not so easy to understand what you are trying to achieve here.
I guess you like to increase the header height if a user is logged in to display an additional element?

Now, `GlobalStyles` depend on the user state. Instead, the row height could be controlled by the content/header (`auto`).